### PR TITLE
GET /releases 

### PR DESCRIPTION
**Pull request by mchalski**
https://tracker.mender.io/browse/MEN-1428

modifications: releases have no `id` or `modified_ts` since they're not real resources yet (aggregated on the fly).

btw i'm using the store layer straight from the controller, it saves some work. the app layer is not necessary yet, add when really required.

### CODE REVIEW

File: docs/management_api.yml [hunk 1]
```diff
@@ -923,16 +1003,63 @@ definitions:
...
     required:
       - limit
       - usage
     example:
       application/json:
         limit: 1073741824
         usage: 536870912
-
+  Release:
+    description: Groups artifacts with the same release name into a single resource.
```
Comment (maciejmrowiec) [inline docs/management_api.yml]: Grouping more of conternal implementation, conceptually this is a stand alone resource.

Comment (mchalski) [reply docs/management_api.yml]: so..."Software release definition"?

Comment (maciejmrowiec) [reply docs/management_api.yml]: Maybe something along the lines:

Release is a group of mender artifacts containing same version of software that targets variety of different platform types.

Comment (mchalski) [reply docs/management_api.yml]: fixed, but used `...different device types` instead of '...platform types' to be 1:1 with field names

**Pull request merged**